.gitignore

```diff
@@ -0,0 +1,74 @@
+*.a
+*.bak
+*.db
+*.flash
+*.gcda
+*.gcno
+*.log
+*.o
+*.lo
+*.la
+*.ninja*
+*.opendb
+*.orig
+*.pcap
+*.pyc
+*.suo
+*.swn
+*.swo
+*.swp
+*.trs
+*.user
+*.bak
+*~
+.deps
+.dirstamp
+.DS_Store
+.local-version
+.libs
+.vagrant
+aclocal.m4
+autom4te.cache
+build
+CMakeCache.txt
+CMakeFiles
+cmake_install.cmake
+configure
+config.log
+config.status
+doc/Doxyfile
+doc/html
+etc/cmake/openthread-config-generic.h
```

Comment:
A number of these files are not applicable to this repo. Should we remove them from here?

Reply:
cleaned up those openthread-only ignores, PTAL.